packages/core/src/codewhisperer/util/commonUtil.ts

```diff
@@ -30,6 +30,17 @@ export function asyncCallWithTimeout<T>(asyncPromise: Promise<T>, message: strin
     })
 }
 
+export function asyncCallWithTimeoutSwallowErrors<T>(asyncPromise: Promise<T>, timeLimit: number): Promise<T> {
```

Comment:
maybe `tryTimeout`? This could be moved to a shared utility file or class.

Reply:
moved to timeoutUtils.ts

Reply:
also renamed